<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Proyecto1</class>
 <widget class="QMainWindow" name="Proyecto1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget(
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(255, 170, 127, 255), stop:1 rgba(255, 255, 255, 255));
qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0))
qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(229, 170, 255, 255), stop:1 rgba(255, 255, 255, 255)))</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 85, 255);
color: rgb(255, 255, 255);
font: 75 italic 12pt &quot;Bodoni MT&quot;;
font: 75 italic 12pt &quot;Bodoni MT&quot;;
color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
color: rgb(85, 170, 127);
background-color: rgb(0, 0, 0);
font: 75 12pt &quot;Arial Narrow&quot;;</string>
    </property>
    <property name="text">
     <string>Cargar Archivo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
color: rgb(255, 255, 255);
color: rgb(85, 255, 255);
font: 75 italic 12pt &quot;Bodoni MT&quot;;
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Cargar Opcion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>320</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255);
color: rgb(85, 255, 255);
font: 75 italic 12pt &quot;Bodoni MT&quot;;
background-color: rgb(0, 0, 0);
font: 63 12pt &quot;Yu Gothic UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Analizar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;Baskerville Old Face&quot;;
font: 18pt &quot;MS Shell Dlg 2&quot;;
font: 75 italic 20pt &quot;Georgia&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Generador HTML</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 255, 255);
color: rgb(255, 255, 255);
font: 12pt &quot;Elephant&quot;;</string>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Reporte de tokens</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Reporte de errores</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Manual de Usuario</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Manual Tecnico</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
